cadquery/cq.py

```diff
@@ -2467,6 +2472,8 @@ def eachpoint(
             for o in self.objects:
                 if isinstance(o, (Vector, Shape)):
                     pnts.append(loc.inverse * Location(plane, o.Center()))
+                elif isinstance(o, Sketch):
+                    pnts.append(loc.inverse * Location(plane, o._faces.Center()))
```

Comment:
I'm struggling to understand the desired behaviour here, can you give an example of when you would use `eachpoint` with a sketch on the stack?

Reply:
It is just a simple generalization of the current behavior. If something is possible and fits in the current framework, I wan to enable it.

Reply:
OK then, this will work fine.